.gitignore

```diff
@@ -8,6 +8,7 @@ __pycache__/
 *.so
 
 # Distribution / packaging
+tastyworks/documentation
```

Comment:
Saved data locally for documentation that I don't want to commit

Reply:
As below, we don't want to couple this code to individual use-cases

Reply:
Got it, I think this was in order to include an example on how to do that but this can be removed and stick with the lower level list/dict. I will remove this.

Reply:
This is so that my local documentation folder was not placed in the commits by mistakes. I saw other folders like that in the gitignore so I thought it was were that would go. I will see how not to have these in commits another way

Reply:
Removed